examples/lock-app/cc13x2_26x2/README.md

```diff
@@ -0,0 +1,177 @@
+# CHIP CC1352 CC2652 Lock Example Application
+
+An example application showing the use [CHIP][chip] on the Texas Instruments
+CC13X2_26X2 family of Wireless MCUs.
+
+---
+
+-   [CHIP CC1352 CC2652 Lock Example Application](#chip-cc1352-cc2652-lock-example-application)
+    -   [Introduction](#introduction)
+        -   [Device UI](#device-ui)
+    -   [Building](#building)
+        -   [Preparation](#preparation)
+        -   [Compilation](#compilation)
+    -   [Programming](#programming)
+        -   [UniFlash](#uniflash)
+        -   [Code Composer Studio](#code-composer-studio)
+    -   [Viewing Logging Output](#viewing-logging-output)
+
+---
+
+## Introduction
+
+![CC1352R1_LAUNCHXL](doc/images/cc1352r1_launchxl.jpg)
+
+The CC13X2_26X2 lock example application provides a working demonstration of a
+connected door lock device. This uses the open-source CHIP implementation and
+the Texas Instruments SimpleLink™ CC13x2 and CC26x2 software development kit.
+
+This example is enabled to build on all members of the `CC13X2_26X2` family of
+MCUs [recommended for Thread][ti_thread_dnd]. By default this example targets
+the [CC1352R1_LAUNCHXL][cc1352r1_launchxl] LaunchPad.
+
+The lock example is intended to serve both as a means to explore the workings of
+CHIP, as well as a template for creating real products based on the Texas
+Instruments devices.
+
+## Device UI
+
+This example application has a simple User Interface to depict the state of the
+door lock and to control the state. The user LEDs on the LaunchPad are set on
+when the lock is locked, and are set off when unlocked. The LEDs will flash when
+in the transition state between locked and unlocked. The user buttons are used
+for requesting lock and unlock of the door lock. The left button (`BTN-1`) is
+used to request locking. The right button (`BTN-2`) us used to request
+unlocking.
+
+## Building
+
+### Preparation
+
+Some initial setup is necessary for preparing the build environment. This
+section will need to be done when migrating to new versions of the SDK.
+
+-   Download and install the [SimpleLink™ CC13x2 and CC26x2 software development
+    kit (SDK)][simplelink_sdk] ([4.30.00.54][simplelink_sdk_4.30.00.54])
+
+    -   Follow the default installation instructions when executing the
+        installer.
+
+    -   The version of OpenThread used in this repository is newer than the one
+        packaged with the TI SDK. Check the following section for a list of
+        changes needed.
+
+-   Download and install [SysConfig][sysconfig]
+    ([sysconfig-1.5.0_1397][sysconfig-1.5.0_1397])
+
+    -   This may have already been installed with your SimpleLink SDK install
+
+-   Download and install [FreeRTOS][freertos]
+    ([FreeRTOSv10.3.1.zip][freertosv10.3.1])
```

Comment:
This is upstream FreeRTOS ? Would it be to use https://github.com/FreeRTOS/FreeRTOS as a submodule?

Reply:
done, I have a personal history of messing up git submodules, so they scare me.